src/Microsoft.SqlTools.ServiceLayer/Utility/FeaturesMetadataProviderHelper.cs

```diff
@@ -11,18 +11,25 @@ namespace Microsoft.SqlTools.ServiceLayer.Utility
 {
     public class FeaturesMetadataProviderHelper
     {
-        public static FeatureMetadataProvider[] CreateFratureMetadataProviders()
+        public static FeatureMetadataProvider[] CreateFeatureMetadataProviders()
         {
-            List<FeatureMetadataProvider> featues = new List<FeatureMetadataProvider>();
+            List<FeatureMetadataProvider> features = new List<FeatureMetadataProvider>();
 
-            featues.Add(new FeatureMetadataProvider
+            features.Add(new FeatureMetadataProvider
             {
                 FeatureName = "Restore",
                 Enabled = true,
                 OptionsMetadata = RestoreOptionsHelper.CreateRestoreOptions()
             });
 
-            return featues.ToArray();
+            features.Add(new FeatureMetadataProvider
+            {
+                FeatureName = "serializationService",
```

Comment:
nit: We should stick to a consistent naming pattern for these. Based off of "Restore" above, I'd capitalize this as "SerializationService" or just call it "Serialization"